test/helpers.ts

```diff
@@ -0,0 +1,230 @@
+// import { RedisClient } from 'redis';
+// import type { Socket } from 'socket.io-client';
+// import socketio from 'socket.io-client';
+import { EventEmitter } from 'events';
+import http from 'http';
+
+import socketio, { Socket } from 'socket.io-client';
+import WebSocket from 'ws';
+
+import { CredentialsManager } from '../src/CredentialsManager';
+import RedisSubscriber from '../src/RedisSubscriber';
+import type SubscriptionServer from '../src/SubscriptionServer';
+
+function uuid() {
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, (c) => {
+    // eslint-disable-next-line no-bitwise
+    const r = (Math.random() * 16) | 0;
+    // eslint-disable-next-line no-bitwise
+    const v = c === 'x' ? r : (r & 0x3) | 0x8;
+    return v.toString(16);
+  });
+}
```

Comment:
is this really needed? why not just

```ts
let _id = 0;
function id = () => id++;
```

it's that this pr already adds so much code, and these lines are pretty intellegible

Reply:
I copied this over the other PR removed now